contract-ui/tabs/overview/components/NFTDetails.tsx

```diff
@@ -1,26 +1,44 @@
 import { Flex, useBreakpointValue } from "@chakra-ui/react";
-import { useNFTs } from "@thirdweb-dev/react";
-import { SmartContract } from "@thirdweb-dev/sdk";
 import { SupplyCards } from "contract-ui/tabs/nfts/components/supply-cards";
 import { useTabHref } from "contract-ui/utils";
 import { Heading, TrackedLink, TrackedLinkProps } from "tw-components";
 import { NFTCards } from "./NFTCards";
+import { thirdwebClient } from "components/app-layouts/providers";
```

Comment:
if we *are* going to import it everywhere probably makes sense to define it in a special file